crates/jstzd/src/protocol/mod.rs

```diff
@@ -60,6 +64,8 @@ pub struct ProtocolParameterBuilder {
     protocol: Option<Protocol>,
     /// Protocol constants.
     constants: Option<ProtocolConstants>,
+    /// Bootstrap accounts.
+    bootstrap_accounts: BootstrapAccounts,
     /// Path to an existing parameter file whose content will be used as the base
     /// parameter set. Optional. If `source_path` is not given, a predefined parameter
     /// file will be used instead depending on `protocol` and `constants`.
```

Comment:
Sorry, I missed this in the previous PR

```suggestion
    /// Optional path to an existing parameter file whose content will be used as the base
    /// parameter set. If `source_path` is not given, a predefined parameter
    /// file will be used instead depending on `protocol` and `constants`.
```